### PR TITLE
Improvements to minor mistakes in documentation comments

### PR DESCRIPTION
**Pull request by ANF-Studios**
This PR addresses some mistakes in the documentation comments. No code logic has been changed, only grammatical and punctuation improvements, deletion of unnecessary words. These commits are also gpg signed.

Let me know if you have any objections/questions, I'll work on it as soon as possible.

### CODE REVIEW

File: imgui.cpp [hunk 1]
```diff
@@ -157,13 +156,11 @@ CODE
 
  HOW TO UPDATE TO A NEWER VERSION OF DEAR IMGUI
  ----------------------------------------------
- - Overwrite all the sources files except for imconfig.h (if you have made modification to your copy of imconfig.h)
- - Or maintain your own branch where you have imconfig.h modified as a top-most commit which you can regularly rebase over master.
+ - Overwrite all the sources files except for imconfig.h (if you have modified your copy of imconfig.h)
+ - Or maintain your own branch where you have imconfig.h modified as a top-most commit which you can regularly rebase overmaster.
```
Comment (eliasdaler) [inline imgui.cpp:160]: Should be "over master" or "over master branch" for clarity, if you will.

**Review by eliasdaler — COMMENTED** [review]
One small comment, otherwise looks good - improves clarity of some sentences, which is great.

**Comment by ocornut** [comment]
Hello,
Thanks for the PR! 
I have now squashed this, added minor additions (reported some of your changes in FAQ.md) and merged.

File: imgui.cpp (hunk 1 above)
Comment (ANF-Studios) [reply imgui.cpp:160]: Oh, right, yea. My bad :sweat_smile:
This was modified in the merge, so I'll resolve this conversation.

Reference: https://github.com/ocornut/imgui/commit/ebe6ac5fb55aacb2a3cb0d7ee0bef7cba89cac1e#diff-cee9beb740469eb5192dfe94aaee7320932590adade9d9b1d2ee392ac2e1e88cR161

**Comment by ANF-Studios** [comment]
Great! Glad I could contribute to what I use 😄

**Pull request closed without merging**